modules/content/.gitignore

```diff
@@ -0,0 +1,11 @@
+# dependencies
```

Comment:
Why is this .gitignore needed? If additional ignores required we may just move them into project root's .gitignore